python/hsfs/feature_view.py

```diff
@@ -105,6 +105,21 @@ def init_serving(
         )
         self._vector_server.init_serving(self, batch, external)
 
+    def init_batch_scoring(
+        self,
+        training_dataset_version: Optional[int] = None,
+    ):
+        """Initialise and cache parametrized transformation functions.
+
+        # Arguments
+            training_dataset_version: int, optional. Default to be 1. Transformation statistics
+                are fetched from training dataset and apply in serving vector.
+        """
+        self._vector_server = vector_server.VectorServer(
+            self._featurestore_id, training_dataset_version
+        )
+        self._vector_server.init_batch_scoring(self)
```

Comment:
You may need 2 `_vector_server` instance, because `get_feature_vector` won't work if users call `init_batch_scoring` first then call `get_feature_vector`

Reply:
yes thats, but its similar what happens if users call  `init_serving(batch=True)` and then they call `get_feature_vector`. We need explain this well in the docs